Frameworks/Foundation/NSURLSession.mm

```diff
@@ -761,15 +757,21 @@ - (void)downloadTask:(NSURLSessionDownloadTask*)task
                              totalBytesExpectedToWrite);
 }
 
+- (void)_destroyTemporaryFile:(NSURL*)url {
+    [[NSFileManager defaultManager] removeItemAtURL:url error:nullptr];
+}
+
 - (void)downloadTask:(NSURLSessionDownloadTask*)task didFinishDownloadingToURL:(NSURL*)url {
     _NSURLSessionInflightTaskInfo* inflightTaskData = [self _inflightDataForTask:task];
     if (inflightTaskData) {
         [inflightTaskData dispatchDownloadCompletionHandlerForURL:url];
+        [self _destroyTemporaryFile:url];
     } else {
         // NOT OPTIONAL (protocol specifies @required)
         auto properDelegate(reinterpret_cast<id<NSURLSessionDownloadDelegate>>(_delegate));
         [_delegateQueue addOperationWithBlock:^{
             [properDelegate URLSession:self downloadTask:task didFinishDownloadingToURL:url];
+            [self _destroyTemporaryFile:url];
         }];
     }
```

Comment:
the changes.

looks like precommit clang script  decided to format it it's way